indico/modules/events/static/templates/static_sites.html

```diff
@@ -0,0 +1,64 @@
+<div class="i-box-group vert fixed-width static-sites">
+    <div class="i-box titled">
+        <div class="i-box-header">
+            <div class="i-box-title">{%- trans %}New Offline Copy{% endtrans -%}</div>
+            <button id="test" class="i-button icon-wrench right" type="button"
+                    data-method="post"
+                    data-href="{{ url_for('static_site.build', event) }}"
+                    data-title="{{ _('Build an offline copy') }}"
+                    data-confirm="{{ _('Are you sure you want to build an offline copy of this event?<br>Beware that it is a heavy operation, especially for large events, and it might take some time to finish.') }}">{{_('Build offline copy')}}
+            </button>
+        </div>
+        <div class="i-box-content">
+            {% trans %}This will build an offline copy of the event. This is useful if you want to make it available where no Internet connection is available.{% endtrans %}</li>
+        </div>
+    </div>
+    <div class="i-box titled no-padding">
+        <div class="i-box-header">
+            <div class="i-box-title">{%- trans %}Existing Offline Copies{% endtrans -%}</div>
+        </div>
+        <div class="i-box-content">
+            {% if static_sites %}
+                {% set label_mapping = {
+                    'success': 'accept',
+                    'failed': 'danger',
+                    'expired': 'warning',
+                    'running': 'highlight'
+                } %}
+                {% set has_downloads = static_sites|selectattr('state.name', 'equalto', 'success')|list|count %}
```

Comment:
`app.add_template_filter(any)` in indico/web/flask/app.py:setup_jinja would allow you to use this:

    {% set has_downloads = static_sites|selectattr('state.name', 'equalto', 'success')|any %}